cedar-policy/src/api.rs

```diff
@@ -308,6 +393,29 @@ impl Entities {
     /// If a `schema` is provided, this will inform the parsing: for instance, it
     /// will allow `__entity` and `__extn` escapes to be implicit, and it will error
     /// if attributes have the wrong types (e.g., string instead of integer).
+    /// ```
+    /// use std::collections::HashMap;
+    /// use std::str::FromStr;
+    /// # use cedar_policy::{Entities, EntityId, EntityTypeName, EntityUid, EvalResult, Request,PolicySet};
```

Comment:
should we hide all of these imports, not just the `cedar_policy` ones?
```suggestion
    /// # use std::collections::HashMap;
    /// # use std::str::FromStr;
    /// # use cedar_policy::{Entities, EntityId, EntityTypeName, EntityUid, EvalResult, Request,PolicySet};
```